.github/workflows/ci_docs.yml

```
@@ -18,13 +18,22 @@ name: Docs
 on:
   push:
     branches: [ main ]
+    paths:
+      - 'pygmt/**/*.py'
+      - '!pygmt/tests/**'
+      - 'doc/**'
+      - 'examples/**'
+      - 'README.rst'
+      - '.github/workflows/ci_docs.yml'
```

Maybe add `ci/requirements/docs.yml` too
```suggestion
      - '.github/workflows/ci_docs.yml'
      - 'ci/requirements/docs.yml'
```

---

`ci/requirements/docs.yml` is only used by ReadTheDocs, so it's not related to this workflow.

---

Oh right, ignore this then.